Library/Homebrew/cmd/test-bot.rb

```diff
@@ -520,7 +521,7 @@ def formula formula_name
               test "brew", "uninstall", "--force", *unchanged_build_dependencies
               unchanged_dependencies -= unchanged_build_dependencies
             end
-            test "brew", "install", bottle_filename
+            test "brew", "install", "--force-bottle", bottle_filename
```

Comment:
I don't think it does but I think this behaviour probably shouldn't be the case when installing from a local bottle path CC @dunn and/or should only check for uncommitted changes.